src/app/products/manage-tax-components/create-tax-component/create-tax-component.component.html

```diff
@@ -0,0 +1,96 @@
+            taxComponentForm.controls.percentage.hasError('max')">
+              Percentage <strong>should</strong> be between one and a hundred
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Debit Account Type</mat-label>
+            <mat-select formControlName="debitAccountType">
+              <mat-option *ngFor="let debitAccountType of debitAccountTypeData" [value]="debitAccountType.id">
+                {{ debitAccountType.value }}
+              </mat-option>
+            </mat-select>
+          </mat-form-field>
+
+          <mat-form-field *ngIf="taxComponentForm.controls.debitAccountType.value">
+            <mat-label>Debit Account</mat-label>
+            <mat-select formControlName="debitAcountId">
+              <mat-option *ngFor="let debitAccount of debitAccountData" [value]="debitAccount.id">
+                {{ debitAccount.name }}
+              </mat-option>
+            </mat-select>
+            <mat-error *ngIf="taxComponentForm.controls.debitAcountId.hasError('required')">
+              Debit account is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Credit Account Type</mat-label>
+            <mat-select formControlName="creditAccountType">
+              <mat-option *ngFor="let creditAccountType of creditAccountTypeData" [value]="creditAccountType.id">
+                {{ creditAccountType.value }}
+              </mat-option>
+            </mat-select>
+          </mat-form-field>
+
+          <mat-form-field *ngIf="taxComponentForm.controls.creditAccountType.value">
+            <mat-label>Credit Account</mat-label>
+            <mat-select formControlName="creditAcountId">
+              <mat-option *ngFor="let creditAccount of creditAccountData" [value]="creditAccount.id">
+                {{ creditAccount.name }}
+              </mat-option>
+            </mat-select>
+            <mat-error *ngIf="taxComponentForm.controls.creditAcountId.hasError('required')">
+              Credit account is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+        <mat-form-field>
+          <mat-label>Start date</mat-label>
+          <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="startDatePicker" required formControlName="startDate">
+          <mat-datepicker-toggle matSuffix [for]="startDatePicker"></mat-datepicker-toggle>
+          <mat-datepicker #startDatePicker></mat-datepicker>
+          <mat-error *ngIf="taxComponentForm.controls.startDate.hasError('required')">
+            Start date is <strong>required</strong>
+          </mat-error>
+        </mat-form-field>
+
+        </div>
+
+      </mat-card-content>
+
+      <mat-card-actions fxLayout="row" fxLayout.xs="column" fxLayoutAlign="center" fxLayoutGap="5px">
+        <button mat-raised-button [routerLink]="['../']">Cancel</button>
```

Comment:
```suggestion
        <button type="button" mat-raised-button [routerLink]="['../']">Cancel</button>
```
If you don't add this, cancel button will also submit the form.